src/components/ManageTeamHealth.vue

```diff
@@ -42,35 +61,81 @@ export default {
     },
 
     reset() {
-      this.$emit('reset');
+      // TODO: Reset i
       this.hasChanged = false;
     },
 
     changeStatus(team, key) {
       this.hasChanged = false;
-      if (team.criteria_values[key] === 3) {
-          team.criteria_values[key] = 1;
+      if (team.criteria_values[key] === STATUS.GREEN) {
+          team.criteria_values[key] = STATUS.RED;
       }
       else {
           team.criteria_values[key]++;
       }
       this.hasChanged = true;
     },
+
+    /**
+     * Compares a team's criteria value with the previous sprint value, if it exists, 
+     * to determine whether there was an increase or decrease. 
+     * @returns -1 for decrease, 0 for no change, 1 for increase
+     */
+    getStatusChange(team, key) {
+      // Return 0 if this is the first sprint
+      if (this.currentSprint.sprintNumber === 0) {
+        return 0;
+      }
+
+      // Get the previous sprint
+      const previousSprintNo = parseInt(this.currentSprint.sprintNumber - 1).toString() 
+      const previousSprint = this.getSprintByNumber(previousSprintNo);
+
+      if (!_.isUndefined(previousSprint)) {
```

Comment:
What is the difference between doing this, and just doing !previousSprint? This is more for my learning.

Reply:
`!previousSprint` would also work. The only reason I used `_.isUndefined()` is because `previousSprint` will always be undefined it it doesn't exist. I may change this.